perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -312,6 +312,26 @@ sub inspect : Private {
     $c->stash->{categories} = $c->forward('/admin/categories_for_point');
     $c->stash->{report_meta} = { map { $_->{name} => $_ } @{ $c->stash->{problem}->get_extra_fields() } };
 
+    my @priorities = $c->model('DB::ResponsePriority')->for_bodies($problem->bodies_str_ids)->all;
```

Comment:
Hmm, yes, it should only add the priority that is on all categories, to those categories in the same body as the priority.